packages/labs/test/grpc_web/proto/BUILD.bazel

```diff
@@ -13,6 +13,8 @@ proto_library(
 
 ts_proto_library(
     name = "naming_styles_ts_proto",
+    # Ensure that wire format can change without conflicting outputs.
```

Comment:
I think this only works because it relies on sandboxing.
If you run with `--spawn_strategy=standalone` you'll probably see a failure here, same as the windows failure on CI